### ISSUE TITLE
Reset password does not work

### ISSUE THREAD

**Issue by jnehring** [multidimensionalquality/qt21-scorecard#40]
The link in the password recovery mail starts with

http://lnv-3209.sb.dfki.de/

which is only accessible from inside DFKI.

**Comment by jnehring** [comment]
A workaround for the bug: Exchange the hostname of the link. An example of an activation link is

http://lnv-3209.sb.dfki.de/scorecard/resetting/reset/FggN8YRTP6GGqU8_nwcbFjVQs75-3kWOexWT7q8t_X

Exchange `lnv-3209.sb.dfki.de` with `qt21.eu` and the link works fine.

The problem is the proxy redirect from a server that hosts qt21.eu to lnv-3209.sb.dfki.de . The system thinks that all HTTP requests are directed to lnv-3209.sb.dfki.de but this is only the adress of the underlying server that is not accessible from the outside.

There is no clean solution to fix this with Symfony configuration. I think the easiest way to fix it is

* Copy the routing configuration from FOSUSerBundle to ScorecardBundle
* Create new route compiler and update routing configuration as explained here: http://stackoverflow.com/questions/26783976/set-host-globally-for-routes

So reset password works only from within DFKI network.

**Comment by arlelommel** [comment]
It seems this does actually work when installed on a non-DFKI server.

**Comment by jnehring** [comment]
> It seems this does actually work when installed on a non-DFKI server.

Exactly. The problem originates from the special proxy redirect settings in the DFKI installation.
